modules/microshift-firewall-add-services.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-firewall.adoc
+
+:_mod-docs-content-type: Procedure
+[id="microshift-firewall-add-services_{context}"]
+= Adding services to open ports
+
+On an {microshift-short} instance, you can open services on ports using the `firewall-cmd` command. 
```

Comment:
@pacevedom Do you think this looks clearer?

Also I believe these updates would still apply to versions 4.14 and 4.15, is that correct?

Reply:
That is correct, looking good now!